src/main/java/racingcar/controller/RacingGame.java

```diff
@@ -0,0 +1,98 @@
+package racingcar.controller;
+
+import camp.nextstep.edu.missionutils.Console;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+import racingcar.domain.Car;
+import racingcar.domain.NumberGenerator;
+import racingcar.view.UserInput;
+import racingcar.view.UserOutput;
+
+public class RacingGame {
+    private static RacingGame instance; // 게임을 진행할 싱글톤 인스턴스
+    private final List<Car> cars;
+
+    private RacingGame() {
+        cars = new ArrayList<>();
+    }
+
+    private static RacingGame getInstance() {
+        if (instance == null) {
```

Comment:
```suggestion
        if (Objects.isNull(instance)){
```

java에서 제공하는 API를 활용해보시면 좋을 것 같아 슬쩍 적고 갑니다!